style.css

```
@@ -51,6 +51,16 @@ body {
   padding-left: 15px;
 }
 
+.tweet-block {
+  margin-top: 30px;
+}
+
+.game-over .tweet-icon {
```

```suggestion
.tweet-icon {
```

currently it does nothing as the whole tweet UI is outside game-over div.

---

okay removed